### ISSUE TITLE
Issue with opening files [BUG]

### ISSUE THREAD

**Issue by ccnawrocki** [chanzuckerberg/cellxgene#2633]
Hi, 
I am having trouble opening my files. I have followed the instructions precisely for the installation. I made a conda environment and then installed cellxgene with `pip install cellxgene`. Nonetheless, I keep getting this error. 

```
(cellxgene-env) cnawrocki@carbunculus ~ % cellxgene launch PC30_cellxgene.h5ad --open
Traceback (most recent call last):
  File "__init__.pxd", line 942, in numpy.import_array
RuntimeError: module compiled against API version 0x10 but this version of numpy is 0xf

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/cnawrocki/anaconda3/envs/cellxgene-env/bin/cellxgene", line 5, in <module>
    from server.cli.cli import cli
  File "/Users/cnawrocki/anaconda3/envs/cellxgene-env/lib/python3.11/site-packages/server/cli/cli.py", line 3, in <module>
    from .annotate import annotate
  File "/Users/cnawrocki/anaconda3/envs/cellxgene-env/lib/python3.11/site-packages/server/cli/annotate.py", line 12, in <module>
    import pandas as pd
  File "/Users/cnawrocki/anaconda3/envs/cellxgene-env/lib/python3.11/site-packages/pandas/__init__.py", line 22, in <module>
    from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cnawrocki/anaconda3/envs/cellxgene-env/lib/python3.11/site-packages/pandas/compat/__init__.py", line 18, in <module>
    from pandas.compat.numpy import (
  File "/Users/cnawrocki/anaconda3/envs/cellxgene-env/lib/python3.11/site-packages/pandas/compat/numpy/__init__.py", line 4, in <module>
    from pandas.util.version import Version
  File "/Users/cnawrocki/anaconda3/envs/cellxgene-env/lib/python3.11/site-packages/pandas/util/__init__.py", line 2, in <module>
    from pandas.util._decorators import (  # noqa:F401
  File "/Users/cnawrocki/anaconda3/envs/cellxgene-env/lib/python3.11/site-packages/pandas/util/_decorators.py", line 14, in <module>
    from pandas._libs.properties import cache_readonly
  File "/Users/cnawrocki/anaconda3/envs/cellxgene-env/lib/python3.11/site-packages/pandas/_libs/__init__.py", line 13, in <module>
    from pandas._libs.interval import Interval
  File "pandas/_libs/interval.pyx", line 1, in init pandas._libs.interval
  File "pandas/_libs/hashtable.pyx", line 1, in init pandas._libs.hashtable
  File "pandas/_libs/missing.pyx", line 1, in init pandas._libs.missing
  File "/Users/cnawrocki/anaconda3/envs/cellxgene-env/lib/python3.11/site-packages/pandas/_libs/tslibs/__init__.py", line 36, in <module>
    from pandas._libs.tslibs import dtypes
  File "pandas/_libs/tslibs/dtypes.pyx", line 1, in init pandas._libs.tslibs.dtypes
  File "pandas/_libs/tslibs/np_datetime.pyx", line 28, in init pandas._libs.tslibs.np_datetime
  File "__init__.pxd", line 944, in numpy.import_array
ImportError: numpy.core.multiarray failed to import
```

Here are the packages installed in the environment: 

```
(cellxgene-env) cnawrocki@carbunculus ~ % conda list
# packages in environment at /Users/cnawrocki/anaconda3/envs/cellxgene-env:
#
# Name                    Version                   Build  Channel
aniso8601                 9.0.1                    pypi_0    pypi
anndata                   0.9.2                    pypi_0    pypi
boto3                     1.28.50                  pypi_0    pypi
botocore                  1.31.50                  pypi_0    pypi
brotli                    1.1.0                    pypi_0    pypi
bzip2                     1.0.8                h620ffc9_4  
ca-certificates           2023.08.22           hca03da5_0  
cellxgene                 1.1.2                    pypi_0    pypi
certifi                   2023.7.22                pypi_0    pypi
charset-normalizer        3.2.0                    pypi_0    pypi
click                     8.1.7                    pypi_0    pypi
flask                     2.2.5                    pypi_0    pypi
flask-compress            1.14                     pypi_0    pypi
flask-cors                4.0.0                    pypi_0    pypi
flask-restful             0.3.10                   pypi_0    pypi
flask-server-timing       0.1.2                    pypi_0    pypi
flask-talisman            1.1.0                    pypi_0    pypi
flatbuffers               1.12                     pypi_0    pypi
flatten-dict              0.4.2                    pypi_0    pypi
fsspec                    0.7.4                    pypi_0    pypi
gunicorn                  21.2.0                   pypi_0    pypi
h5py                      3.9.0                    pypi_0    pypi
idna                      3.4                      pypi_0    pypi
itsdangerous              2.1.2                    pypi_0    pypi
jinja2                    3.1.2                    pypi_0    pypi
jmespath                  1.0.1                    pypi_0    pypi
libffi                    3.4.4                hca03da5_0  
llvmlite                  0.40.1                   pypi_0    pypi
markupsafe                2.1.3                    pypi_0    pypi
natsort                   8.4.0                    pypi_0    pypi
ncurses                   6.4                  h313beb8_0  
numba                     0.57.1                   pypi_0    pypi
numpy                     1.22.0                   pypi_0    pypi
openssl                   3.0.10               h1a28f6b_2  
packaging                 23.1                     pypi_0    pypi
pandas                    1.5.1                    pypi_0    pypi
pip                       23.2.1          py311hca03da5_0  
python                    3.11.5               hb885b13_0  
python-dateutil           2.8.2                    pypi_0    pypi
pytz                      2023.3.post1             pypi_0    pypi
pyyaml                    6.0.1                    pypi_0    pypi
readline                  8.2                  h1a28f6b_0  
requests                  2.31.0                   pypi_0    pypi
s3fs                      0.4.2                    pypi_0    pypi
s3transfer                0.6.2                    pypi_0    pypi
scipy                     1.11.2                   pypi_0    pypi
setuptools                68.0.0          py311hca03da5_0  
six                       1.16.0                   pypi_0    pypi
sqlite                    3.41.2               h80987f9_0  
tk                        8.6.12               hb8d0fd4_0  
tzdata                    2023c                h04d1e81_0  
urllib3                   1.26.16                  pypi_0    pypi
werkzeug                  2.3.7                    pypi_0    pypi
wheel                     0.38.4          py311hca03da5_0  
xz                        5.4.2                h80987f9_0  
zlib                      1.2.13               h5a0b063_0  
```

Any help would be greatly appreciated.

**Comment by mbuttner** [comment]
I am having the same issue with Python 3.11. I assume that downgrading to Python 3.9 will solve the issue.

**Comment by ccnawrocki** [comment]
I made a new conda environment with python 3.9 and retried and it worked. Thank you!

**Comment by philmcnamara** [comment]
I ran into the same issue, also fixed by switching python 3.11 to 3.9. Worth considering updating the README to specify required python version, since the error message makes it seem like a numpy problem
